e2e/playwright/test-utils.ts

```diff
@@ -420,7 +421,7 @@ export async function getUtils(page: Page, test_?: typeof test) {
       const overlay = page.locator(locator)
       const bbox = await overlay
         .boundingBox({ timeout: 5_000 })
-        .then((box) => ({ ...box, x: box?.x || 0, y: box?.y || 0 }))
+        .then((box: any) => ({ ...box, x: box?.x || 0, y: box?.y || 0 }))
```

Comment:
Weird that we lost the type inference on this, do you know why?

Reply:
I have zero idea. I really tried on this one too (relative to others) but after about 10 minutes I gave up.